src/mat4.js

```diff
@@ -1513,7 +1512,7 @@ export function targetTo(out, eye, target, up) {
   out[13] = eyey;
   out[14] = eyez;
   out[15] = 1;
-  return out;
+  return invert(out, out);
```

Comment:
As stated in #225 , this does not need to be inverted. If you want the inverse, you should use `lookAt`

Reply:
ok, due to my poor English, I still can't understand #225 said, it seems lookAt function has achieved the goal which I thought target should would do, Thank you!

Reply:
@jack-shanghai Yes, that is correct. 

Would you mind changing this pull request from `fix: targetTo..` to `optim: Math.hypot..`?

Reply:
sorry, I forget it, it's done